src/pages/FarmV4/FarmItems.tsx

```diff
@@ -118,21 +114,6 @@ const FarmItems: FC<{
         openPositionsAcrossPrograms={openPositionsAcrossPrograms}
       />
       }
-      {((!(numberOfTokensDeposited === 0 && showDeposited) || filteredPools.length !== 0)
```

Comment:
By removing this the Grid view (Lite Mode) won't ever be able to load more in - should this be kept in and only rendered for lite mode, or is the other work for Lite Mode in progress?